Dockerfile

```diff
@@ -212,6 +213,19 @@ RUN mkdir -p /vmtoolsd/${LIBDNET} &&\
 RUN cd $ROOTFS && cd usr/local/lib && ln -s libdnet.1 libdumbnet.so.1 &&\
     cd $ROOTFS && ln -s lib lib64
 
+# Build XenServer Tools
+ENV XENTOOLS_REPO xe-guest-utilities
+ENV XENTOOLS_BRANCH boot2docker
+
+RUN cd / && \
+    git clone https://github.com/xenserver/$XENTOOLS_REPO && \
+    cd $XENTOOLS_REPO && \
+    git checkout $XENTOOLS_BRANCH && \
```

Comment:
Can we adjust this clone/checkout process to mimic AUFS and open-vm-tools? (https://github.com/boot2docker/boot2docker/blob/c011a58c89f746c25d9a989d0b8bc6145e1acdcd/Dockerfile#L183-L189)

ie:
```Dockerfile
ENV XEN_REPO https://github.com/xenserver/xe-guest-utilities
ENV XEN_BRANCH boot2docker
ENV XEN_COMMIT 4a9417fa61a5ca46676b7073fdb9181fe77ba56e

RUN git clone -b "$XEN_BRANCH" "$XEN_REPO" /vmtoolsd \
    && cd /vmtoolsd \
    && git checkout -q "$XEN_COMMIT" \
    && make \
    && tar xvf build/dist/${XENTOOLS_REPO}_*.tgz -C $ROOTFS/
```